squad-server/plugins/cbl-info.js

```diff
@@ -117,11 +114,19 @@ export default class CBLInfo extends DiscordBasePlugin {
           },
           description: `[${info.player.name}](https://communitybanlist.com/search/${info.player.steamID}) has ${data.steamUser.reputationPoints} reputation points on the Community Ban List and is therefore a potentially harmful player.`,
           fields: [
+            {
+              name: 'Steam ID',
+              value: `[${info.player.steamID}](https://steamcommunity.com/profiles/${info.player.steamID})`,
+              inline: true
+            },
+            {
+              name: 'EOS ID',
+              value: info.player.eosID || 'Unknown',
+              inline: true
+            },
             {
               name: 'Reputation Points',
-              value: `${data.steamUser.reputationPoints} (${
-                data.steamUser.reputationPointsMonthChange || 0
-              } from this month)`,
+              value: `${data.steamUser.reputationPoints}`,
```

Comment:
Is there a particular reason for removing the amount of points from the current month? That seems like a useful statistic to establish how recent their poor behaviour is.

Reply:
Please could you apply that change? Unfortunately, it's not easy for me to edit other's pull requests.

Reply:
Never mind, I managed it somehow.